app/repositories/raw_data_record_repository.rb

```diff
@@ -1,10 +1,20 @@
 require 'ostruct'
 require 'register_sources_psc/repositories/company_record_repository'
+require 'register_sources_sk/repositories/record_repository'
+require 'register_sources_dk/repositories/deltagerperson_repository'
 require 'register_sources_bods/structs/identifier'
 
 class RawDataRecordRepository
   def initialize
-    @repository = RegisterSourcesPsc::Repositories::CompanyRecordRepository.new
+    @psc_repository = RegisterSourcesPsc::Repositories::CompanyRecordRepository.new
+    @sk_repository = RegisterSourcesSk::Repositories::RecordRepository.new
+    @dk_repository = RegisterSourcesDk::Repositories::DeltagerpersonRepository.new
+
+    @repositories = [
```

Comment:
Not sure how these are being used, but I wonder about using a hash/dict instead.